code/modules/clothing/outfits/ert.dm

```diff
@@ -460,3 +460,47 @@
 		/obj/item/gun/ballistic/automatic/sniper_rifle=1,\
 		/obj/item/gun/grenadelauncher/security=1,\
 		/obj/item/gun/ballistic/automatic/ar=1)
+
+/datum/outfit/hitman
+	name = "Hitman"
+
+	uniform = /obj/item/clothing/under/suit/black
+	shoes = /obj/item/clothing/shoes/sneakers/black
+	gloves = /obj/item/clothing/gloves/color/black
+	ears = /obj/item/radio/headset
+	glasses = /obj/item/clothing/glasses/sunglasses/advanced
+	l_hand = /obj/item/storage/secure/briefcase
+	id = /obj/item/card/id/syndicate
+	belt = /obj/item/pda/heads
+	r_hand = /obj/item/storage/box/syndie_kit/chameleon
+	r_pocket = /obj/item/kitchen/knife/combat
+
+/datum/outfit/hitman/post_equip(mob/living/carbon/human/H, visualsOnly = FALSE)
+	var/obj/item/clothing/under/U = H.w_uniform
+	U.attach_accessory(new /obj/item/clothing/accessory/waistcoat(H))
+
+	if(visualsOnly)
+		return
+
+	//Could use a type
+	var/obj/item/storage/secure/briefcase/sec_briefcase = H.get_item_for_held_index(1)
+	for(var/obj/item/briefcase_item in sec_briefcase)
+		qdel(briefcase_item)
+	for(var/i = 3 to 0 step -1)
+	SEND_SIGNAL(sec_briefcase, COMSIG_TRY_STORAGE_INSERT, new /obj/item/stack/spacecash/c1000, null, TRUE, TRUE)
+	SEND_SIGNAL(sec_briefcase, COMSIG_TRY_STORAGE_INSERT, new /obj/item/pen/sleepy, null, TRUE, TRUE)
```

Comment:
a sleepypen isnt obvious at first. you should add a note into the briefcase so they know what it is. Honestly, they should have a satchel or smth instead of a briefcase- so they wont be disarmed, yknow?